### DISCUSSION TITLE
Replacement for Fabian?

### DISCUSSION THREAD

**Discussion by smerkel** [dmi-3D/dmi-3D.github.io#36] [Data visualization]
Hi,

As an old folk from the fable community, I still use fabian to visualize diffraction data, peaks, etc, but fabian is getting old. 

Does anyone have a suggestion for a replacement that can visualize both images and peaks extracted from peaksearch. 

Can you do this with silx for instance?

Best,

Sébastien

**Comment by jonwright** [comment]
Hi Sébastien

There are a few things done in an attempt to use the new 'silx' library here:
https://github.com/jonwright/ImageD11/blob/master/ImageD11/silxGui/silx_sptview.py

In practice, we normally use python notebooks now. Then it is just matplotlib with imshow (or pcolormesh) to display the image together with a scatter plot for the peaks. With jupyter it seems to be easier to get a machine on a remote cluster and get the local window onto a screen

Best

Jon

**Comment by smerkel** [comment]
I see. We'll have to look into a quick python notebook or your silx interface. Thanks!
